components/script/script_thread.rs

```diff
@@ -4059,11 +4068,8 @@ impl Drop for ScriptThread {
 }
 
 fn dom_last_modified(tm: &SystemTime) -> String {
-    let tm = tm.duration_since(SystemTime::UNIX_EPOCH).unwrap();
-    let tm = Timespec::new(tm.as_secs() as i64, 0);
-    let tm = at_utc(tm);
-    tm.to_local()
-        .strftime("%m/%d/%Y %H:%M:%S")
-        .unwrap()
-        .to_string()
+    let tm: Duration = tm.duration_since(SystemTime::UNIX_EPOCH).unwrap();
+    let dt = NaiveDateTime::from_timestamp_opt(tm.as_secs() as i64, tm.subsec_nanos()).unwrap();
+    let dt: DateTime<Local> = Local.from_utc_datetime(&dt);
+    dt.format("%m/%d/%Y %H:%M:%S").to_string()
```

Comment:
Maybe we can improve the name of these variables while we are here. Maybe this?

- `tm` –> `current_time`
- `dt` –> `utc_time`
- `dt` –> `local_time`

Please also properly handle when the unwrap would fail here.

Reply:
Yeah I've been putting `unwrap`s all over the place because I have no visibility on the overall functionality. Would you have any idea how to deal with this specific one?

Reply:
See [previous comment](https://github.com/servo/servo/pull/30639#discussion_r1375412348).